script/rubocop-autocorrect.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# Fixes safe cops automatically and creates a commit for each.
+#
+
+if git add --dry-run --all | grep --quiet .; then
+  echo "Dirty working tree. Please start on a fresh branch."
+  exit 1
+fi
+
+# Refresh todo file so that following commits include only related changes:
+bundle exec rubocop --regenerate-todo
+git commit --all --message "Regenerate Rubocop's TODO file"
```

Comment:
I think that we should add `--no-auto-gen-timestamp` to avoid unnecessary commits and merge conflicts. The timestamp can be seen in the Git commit.

Reply:
I do agree.